### ISSUE TITLE
Faded thread scrolling vs hard message list scrolling

### ISSUE THREAD

**Issue by ChristophWurst** [nextcloud/mail#7186]
### Steps to reproduce

1. Open the app
2. Configure an account
3. Open a (long) message/thread
4. Scroll the message list
5. Scroll the message

### Expected behavior

The designers are happy

### Actual behavior

| Message list scrolling | Thread/message scrolling |
|---|---|
| ![Bildschirmfoto vom 2022-09-06 15-37-31](https://user-images.githubusercontent.com/1374172/188649913-3b055c80-864f-41b7-a432-2371b24f8d5f.png) | ![Bildschirmfoto vom 2022-09-06 15-37-19](https://user-images.githubusercontent.com/1374172/188649916-5b2b83bc-0b65-4eb3-88a3-7dcca49937ee.png) |

### Mail app version

2.0.0 beta 1

**Comment by ChristophWurst** [comment]
^ @jancborchardt @nimishavijay should the *New message* header also have some kind of fade for the list underneath?

**Comment by jancborchardt** [comment]
> ^ @jancborchardt @nimishavijay should the _New message_ header also have some kind of fade for the list underneath?

Yeah, that would indeed be nice! Then it seems less boxy. Would probably be good to do this in the components too? We have a similar issue in the left navigation of Talk with the search bar on top, and in all apps with the settings button in the bottom left. cc @marcoambrosini as well.

**Comment by marcoambrosini** [comment]
I think we should drop the gradient altogether and use a small grey border for the top of scroll containers. The main reason Is that for a fade that looks good, the fade itself needs to take a considerable amount of space. And even then, this is not a widely adopted pattern and might be confusing.

**Comment by jancborchardt** [comment]
I’m fine either way. But it should be consistent, and needs to look good in the newly styled sidebar as well as on color-main-background, both light and dark theme.
